forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-mock-store/forgerock-openbanking-uk-aspsp-rs-mock-store-server/src/main/java/com/forgerock/openbanking/aspsp/rs/store/repository/migration/MongobeeRunnerManually.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2019 ForgeRock AS.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.forgerock.openbanking.aspsp.rs.store.repository.migration;
+
+import com.github.mongobee.Mongobee;
+import com.github.mongobee.exception.MongobeeException;
+import com.mongodb.client.MongoClients;
+import org.springframework.data.mongodb.core.MongoTemplate;
+
+public class MongobeeRunnerManually {
```

Comment:
I buy the suggested name but I'm not confortable to move this local Runner to the test source directory because this means that we need add like a dependency each aspsp module that contains a mongobee ChangeLog distorting the purpose of test support library and increasing possible conflicts due the duplicated dependencies through other projects, I'm more comfortable to delete it than move it to the test support module.

Reply:
I don't understand why you'd need to change the dependencies? I'm talking about moving this from `forgerock-openbanking-uk-aspsp-rs-mock-store-server/src/main` to `forgerock-openbanking-uk-aspsp-rs-mock-store-server/src/test`